nstar/src/main.rs

```diff
@@ -55,10 +55,21 @@ async fn send_request(
     client.request(request).await
 }
 
-async fn create_client(host: &str) -> Result<api::client::Client> {
-    api::client::Client::new(&url::Url::parse(host)?)
-        .await
-        .context(format!("Failed to stablish connection to {}", host))
+const RECONNECT_RETRIES: u64 = 5;
+
+/// Tries to connect to northstar `RECONNECT_RETRIES` times with 1 sec intervals before failing
+async fn try_connect<W: Write>(output: &mut W, url: &url::Url) -> Result<api::client::Client> {
+    for i in 0..RECONNECT_RETRIES {
```

Comment:
What happens after `RECONNECT_RETRIES`? Wouldn't it make sense to retry forever in interactive mode? 

During development it's convenient to have a console funning forever that connects whenever the rt is reachable instead of having it to restart every while and then.

Reply:
We can make it infinite, no problem. I just thought that maybe a limit would make sense.